src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,18 @@
+import './Modal.scss';
+
+type ModalProps = {
+  closeModal: () => void;
+}
+
+const Modal = ({ closeModal }: ModalProps) => {
+  return (
+    <div className='modal-overlay' >
+      <div className='modal' >
+        <p>Sorry! Cell is unavailable for planting.</p>
+        <button className='modal-button' onClick={closeModal} >Close</button>
+      </div>
```

Comment:
Yahh i like that you made this its own component - if we ever needed to we could turn line 11 into a dynamic piece of data being fed in as a prop from wherever its being called.